dask/dataframe/core.py

```diff
@@ -577,6 +577,40 @@ def _aca_agg(self, token, func, aggfunc=None):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def rolling(self, window, min_periods=None, win_type=None):
+        """Provides rolling transformations.
+
+        Parameters
+        ----------
+
+        window: int
+            Size of the moving window. This window must be smaller
+            than the size of the previous partition.
+        min_periods: int, default None
+            Minimum number of observations in window required to have
+            a value (otherwise result is NA).
+        win_type: string, default None
+            Provide a window type. (Identical to pandas.)
+
+        The center, freq, and axis arguments are not supported.
+        """
+        from dask.dataframe.rolling import Rolling
+
+        if not isinstance(window, int):
```

Comment:
Oddly enough, pandas doesn't raise for everything when you call `rolling`, but when you call an actual method sometimes

```
In [138]: df.rolling(-1)
Out[138]: Rolling [window=-1,center=False,axis=0]
```

Nonetheless, I can probably just call the method on the `_pd` too to get the exceptions to bubble..?